src/pages/Pathway/PathwayPDF.js

```diff
@@ -0,0 +1,117 @@
+import { Link, Document, Page, Text, View, StyleSheet, Font } from "@react-pdf/renderer";
+
+const PathwayPDF = ({ resources }) => {
```

Comment:
I haven't seriously looked into it, but I wonder if we can effectively reduce duplicate code by having the `PathDay` component use `View`, `Document`, etc from `@react-pdf/renderer`.

I took a quick look and it seems possible, but I could be wrong. Maybe take a bit of time and if doesn't work easily lmk and I might take a look as well!